sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Will need to move this into a separate directory and have a separate changelog. The necessary files can be pulled in via links.